scripts/verdaccio.sh

```diff
@@ -18,12 +18,10 @@ function cleanup {
   then
     lsof -ti tcp:4000 | xargs kill
     # Clean up generated dists if run locally
-    shopt -s globstar extglob
     rm -rf packages/**/dist
     rm -rf storage/ ~/.config/verdaccio/storage/ $output
     if [ "$commit_to_revert" != "HEAD" ];
     then
-      git tag -d $(git tag -l)
```

Comment:
lerna default tagged everything and created a commit, using yarn, we can simplify and not create tags and not create extra commits